@here/harp-mapview/lib/MapView.ts

```diff
@@ -3422,6 +3464,8 @@ export class MapView extends THREE.EventDispatcher {
             stats.addMemoryInfo();
         }
 
+        this.m_taskScheduler.processPending(frameStartTime);
```

Comment:
Previously at least the geometry creation was outside the animation frame by using setTimeout.
Now we are blocking the animation frame when we process tasks, so I think we should still use setTimeout to call `processPending`

Reply:
ok